Hymns/Browse/TagListView.swift

```diff
@@ -11,11 +11,18 @@ struct TagListView: View {
 
     var body: some View {
         VStack {
+            if viewModel.tags.isEmpty {
+                VStack {
+                    Image("empty tag illustration")
+                    Text("Create tags by tapping on the tag icon on any hymn").padding().multilineTextAlignment(.center)
+                }.frame(minWidth: 0, maxWidth: .infinity, minHeight: 0, maxHeight: .infinity, alignment: .center).background(Color(.systemBackground))
+            } else {
```

Comment:
Nice! Could you create a preview for this state as well? kind of like how I do it here: https://github.com/HymnalDreamTeam/Hymns-iOS/blob/master/Hymns/Display/HymnLyricsView.swift#L56

Reply:
That's possible, but for some reason it works for HymnLyricsView. I tried looking into it, but my previews is broken for some reason and isn't bundling the previews stuff. Can you see why it works for HymnLyricsView but not for this?

Reply:
ok, I figured out why. I found a way to fix it by using .publisher, but we can keep this for now.

Reply:
I was looking at this. I think its because with HymnLyricsView you can feed in the preview to the function since it takes an argument. With TagViewList the function viewModel.getUniqueTags() we can't feed in the results to stub it. Or at least idk how...
        
classic1334ViewModel.lyrics = classic1334ViewModel.convertToViewModels(verses: classic1334_preview.lyrics)